automation/publish_show.py

```diff
@@ -0,0 +1,54 @@
+"""Script to be ran by GH Actions to build the archive and schedule the newsletter"""
+
+import typer
+from src.github import Episode
+from src import newsletter
+from src.engine import engine
+from dateutil import parser
+import pathlib
+
+def build_website(episode:Episode) -> str:
```

Comment:
```suggestion
def build_website(episode:Episode) -> int:
```

Update return type per [docs](https://docs.python.org/3/library/pathlib.html#pathlib.Path.write_text) and [typeshed](https://github.com/python/typeshed/blob/1195bbaf33913292193d5b9d63df04414ede6cf9/stdlib/pathlib.pyi#L193-L198).